Library/ViewModels/DiscoveryNavigationHeaderViewModel.swift

```diff
@@ -210,7 +210,11 @@ public final class DiscoveryNavigationHeaderViewModel: DiscoveryNavigationHeader
       .map { $0.selected ? Strings.discovery_favorite_categories_buttons_unfavorite_a11y_label() :
         Strings.discovery_favorite_categories_buttons_favorite_a11y_label() }
 
-    self.showFavoriteOnboardingAlert = strings.map { $0.subcategory ?? $0.filter }
+    self.showFavoriteOnboardingAlert = strings
+      .map { $0.subcategory != nil ?
+        Strings.category_name_subcategory_name_saved(
+          category_name: $0.filter, subcategory_name: $0.subcategory ??  "") :  $0.filter
+      }
```

Comment:
This condition would be easier to read if the clauses are on separate lines, ala:
```
.map { $0.subcategory != nil
 ? Strings.category_name_subcategory_name_saved(category_name: $0.filter, subcategory_name: $0.subcategory ??  "")
 : $0.filter
}